packages/core/package.json

```diff
@@ -98,6 +98,8 @@
     "it-length-prefixed": "^8.0.2",
     "it-pipe": "^2.0.4",
     "libp2p": "0.40.0",
+    "chai": "^4.3.4",
```

Comment:
I just missed that. `chai` and `fast-check` are dependencies used in test. They should be in `devDependencies`, not `dependencies`. Why were they moved?